### PR TITLE
Refactoring progress

### PR DESCRIPTION
**Pull request by lnabzdyk**
Checkpoint towards final submission. Want to see how things are looking and what needs to be improved before final.

### CODE REVIEW

File: articles/article1.html [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
 </head>
-    
+
     <body>
-        
+
     <header>
-        <div id=sitemasthead> 
-            
+        <div id=sitemasthead>
```
Comment (philschanely) [inline articles/article1.html]: While we're refactoring let's make sure all attributes have quotes around them.

File: sass/_content.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#content {
+    float:none;
+    width:770px;
+    padding:15px 310px 10px 115px;
+    @media screen and (max-width:1200px) {
+            padding:0 25;
```
Comment (philschanely) [inline sass/_content.scss]: Let's also watch for where we need `px` beside units like here on padding. May not be many other places, but this caught my eye.

**Comment by philschanely** [comment]
Clever work refactoring here overall. I've made some picky comments on some files. Review those and commit the requisite updates for further review. 

Great job all around though!

**Comment by lnabzdyk** [comment]
Ready for review of requisite updates!

**Comment by philschanely** [comment]
Approved. Well done! Go ahead and move forward. Drop down the arrow next to the "Merge pull request" and choose "Squash and Merge". Then you'll have the option to provide a commit message that will be used as a single commit to master bringing over all your changes from the develop branch.

**Pull request merged**